src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-your-applications-kubernetes.mdx

```diff
@@ -104,116 +106,81 @@ To link Openshift and Kubernetes you must enable mutating admission webhooks, wh
    ```
 4. Restart the Openshift services in the master node.
 
-## Configure the injection of metadata [#configure-injection]
+## Set up the injection of metadata [#configure-injection]
 
-By default, all the pods you create that include APM agents have the correct environment variables set and the metadata injection applies to the entire cluster. To check that the environment variables have been set, any container that is running must be stopped, and a new instance started (see [Validate the injection of metadata](/docs/integrations/kubernetes-integration/link-your-applications/link-your-applications-kubernetes/#validate-injection)).
+Injection of metadata is included when you [install our integration using Helm](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm). Just be sure that, when you are configuring the chart, the webhook that inject the metadata is enabled:
```

Comment:
"Just be sure that, when you are configuring the chart, the webhook that inject the metadata is enabled"

Is it needed? It is enabled by default

Reply:
resolved